baseclass-samples/java-callouts-baseclass-exception/README.md

```diff
@@ -0,0 +1,75 @@
+# Apigee Java Callouts Base Class Sample - Exception
+
+This sample is a simple proxy that uses a Java callout policy demonstrating
+usage of the Java Callout Base Class.
+
+This sample proxy raises an exception that we store in a flow variable for
+further inspection. It also makes use of the `log` functionality to log a
+message which is also stored in a flow variable for debugging purposes.
+
+## Initial Setup
+
+An Apigee account is required to deploy these sample proxies. You can sign up
+for a free trial account [here.](https://accounts.apigee.com/accounts/sign_up)
+
+You also need to have [Python](http://python.org/getit/) on your system to use
+the deployment tool.
+
+The project is configured to be compiled using
+[Maven](https://maven.apache.org/download.cgi), but `javac` will also work.
+
+To begin, checkout this project to your system.
+
+Next, you must set up your deployment environment:
+
+1. Open the file ../../setup/setenv.sh in a text editor.
```

Comment:
Super nit: Wrap this file name in tick marks so it shows up in the markdown and repeat for other files.